js/back_to_top.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * File back_to_top.js.
+ *
+ * JS for back to top button button.
+ *
+ */
+
+ ( function( $ ) {
+
+	$( window ).scroll(function() {
+
+		var vh = $( window ).height();
```

Comment:
If `var vh = $( window ).height();` is inside the function that fires on scroll, we're recalculating the window height on every scroll event - even though the window size is unlikely to change very often. I logged it to the console and it fired over a 100 times on a simple scroll down my sample page - but the height never changed.

Could we get away with putting the calculation outside the function that's firing on scroll?
It would mean that we only get the height on page load - and a change in window height would cause the 'back to top' link to appear at the wrong time (or not at all if you really shrink the window). I'm just wondering if that's better than running a function to calculate a value that's not going to change too much in regular use.

Or, split the difference and recalculate this only when the window size changes?

Reply:
By the way, everything works just fine and I'm happy to approve. I just noticed this and was wondering if we should change it.